bpf/lib/conntrack.h

```diff
@@ -181,6 +181,16 @@ static __always_inline __u8 __ct_lookup(const void *map, struct __ctx_buff *ctx,
 	entry = map_lookup_elem(map, tuple);
 	if (entry) {
 		cilium_dbg(ctx, DBG_CT_MATCH, entry->lifetime, entry->rev_nat_index);
+
+		if (dir == CT_SERVICE && entry->tx_closing) {
```

Comment:
Just to be future-proof, could you add a small helper:
```
static __always_inline bool ct_entry_closing(const struct ct_entry *entry)
{
	return entry->tx_closing || entry->rx_closing;
}
```

And rework above to: `if (dir == CT_SERVICE && ct_entry_closing(entry))` ?

Otherwise looks great, ack modulo this small refactor above!